### PR TITLE
Create topachat.com.xml

### PR DESCRIPTION
**Pull request by 6u621jpq6ybdw**

### CODE REVIEW

File: src/chrome/content/rules/topachat.com.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!--
+	Connection refused:
+		topachat.com
```
Comment (J0WI) [inline src/chrome/content/rules/topachat.com.xml]: Could you add a redirect from `^http://topachat\.com/` to `https://www.topachat.com/`?

**Comment by 6u621jpq6ybdw** [comment]
@J0WI Added.

**Comment by J0WI** [comment]
You need to add it as target, otherwise the rule is not valid.

**Comment by 6u621jpq6ybdw** [comment]
@J0WI Added.

**Comment by J0WI** [comment]
Please add `<target host="topachat.com" />` and remove the test url.

**Comment by 6u621jpq6ybdw** [comment]
@J0WI oh right the rule must apply to that domain in the first place, ok fixed now and sorry for that!

**Pull request merged**